<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Webcam</string>
   </property>
  </widget>
  <widget class="QLabel" name="camera_display">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="threshhold_display">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Webcam</string>
   </property>
  </widget>
  <widget class="QLabel" name="color2_value">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Setup the Histogram.</string>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="camera_here_text">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera Here:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Threshold_here_text">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold Here:</string>
   </property>
  </widget>
  <widget class="QLabel" name="toggle_text">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Here:</string>
   </property>
  </widget>
  <zorder>start_button</zorder>
  <zorder>stop_button</zorder>
  <zorder>threshhold_display</zorder>
  <zorder>camera_display</zorder>
  <zorder>color2_value</zorder>
  <zorder>text</zorder>
  <zorder>camera_here_text</zorder>
  <zorder>Threshold_here_text</zorder>
  <zorder>toggle_text</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
